runtime/lua/vim/lsp/handlers.lua

```diff
@@ -149,6 +149,28 @@ M['workspace/applyEdit'] = function(_, _, workspace_edit)
   }
 end
 
+--@see https://microsoft.github.io/language-server-protocol/specifications/specification-current/#workspace_configuration
+M['workspace/configuration'] = function(err, _, params, client_id)
+  local client = vim.lsp.get_client_by_id(client_id)
```

Comment:
Should there also be a `nil` check for the `client` similar to how it's done here?

https://github.com/neovim/neovim/commit/fb90b6ec9803fdbd603b1d5f1685583218c18909#diff-728d3ae352b52f16b51a57055a3b20efc4e992efacbf1c34426dfccbba30037cR73